Docs/conf.py

```diff
@@ -36,12 +35,20 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
-    'workaround_recommonmark_issue_191',
-    'recommonmark',
+    'myst_parser',
     'sphinx_markdown_tables',
-    'notfound.extension',  # Show a better 404 page when an invlid address is entered
+    'notfound.extension',  # Show a better 404 page when an invalid address is entered
+]
+
+myst_enable_extensions = [
+    "colon_fence",  # Allow code fence using ::: (see https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#syntax-colon-fence)
+    "linkify",  # Allow automCreate links from URLs (it is sufficient to write https://google.com instead of <https://google.com>)
```

Comment:
```suggestion
    "linkify",  # Allow automatic creation of links from URLs (it is sufficient to write https://google.com instead of <https://google.com>)
```